src/GlobalLoop.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace React\EventLoop;
+
+final class GlobalLoop
+{
+    /**
+     * @internal
+     *
+     * @var LoopInterface
+     */
+    public static $loop;
```

Comment:
Why not always force `GlobalLoop::get`? Making this public could potentially mean someone swaps it out and breaks everything.